common/rpc/interceptor/namespace_validator_test.go

```diff
@@ -600,6 +600,44 @@ func (s *namespaceValidatorSuite) Test_StateValidationIntercept_TokenNamespaceEn
 	}
 }
 
+func (s *namespaceValidatorSuite) Test_Intercept_DescribeHistoryHostRequests() {
```

Comment:
there is Test_Intercept_SearchAttributeRequests which already covers other special cases like 
	case *adminservice.AddSearchAttributesRequest,
		*adminservice.RemoveSearchAttributesRequest,
		*adminservice.GetSearchAttributesRequest,
		*operatorservice.AddSearchAttributesRequest,
		*operatorservice.RemoveSearchAttributesRequest,
		*operatorservice.ListSearchAttributesRequest: